<launch>
    <!--my_rec测试launch文件-->
    <!--开启rec节点进行判断-->
    <node pkg="ucar_cam" type="my_rec_test_trace.py" name="my_rec_test_trace" output="screen" />
    <node pkg="ucar_cam" type="sub_img" name="sub_img" output="screen" />

    <!--后面手动设置为1-->
    <!--启动识别主程序-->
    <param name="rec_state" value="0" />
    <!--是否巡线-->
    <param name="trace_line" value="1" />
    <!--巡线是否开始-->
    <param name="start_trace" value="1" />

    <param name="stop_trace" value="0" />

</launch>